client/src/components/PokemonCard/PokemonCard.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import Move from '../Move/Move';
+import './style.css';
+import CanvasDraw from 'react-canvas-draw';
+
+const PokemonCard = (props) => {
+
+    /* Changes font color based on the pokemon type */
+    const getStyle = (type) => {
+        switch (type) {
+            case null:
+                return {};
+            case "Normal":
+                return { color: "" };
+            case "Fire":
+                return { color: "#ff00ff" };
+            case "Water":
+                return { color: "#0000ff" };
+            case "Grass":
+                return { color: "#00ff00" };
+            case "Electric":
+                return { color: "#ffff00" };
+            case "Psychic":
+                return { color: "#ff00ff" };
+            case "Ice":
+                return { color: "#99ccff" };
+            case "Dragon":
+                return { color: "#ff9933" };
+            case "Dark":
+                return { color: "#cc0099" };
+            case "Fairy":
+                return { color: "#ff99cc" };
+            case "Fighting":
+                return { color: "#cc3300" };
+            case "Flying":
+                return { color: "#66ccff" };
+            case "Poison":
+                return { color: "#cc99ff" };
+            case "Ground":
+                return { color: " #cc7400" };
+            case "Rock":
+                return { color: " #666699" };
+            case "Bug":
+                return { color: " #339933" };
+            case "Ghost":
+                return { color: " #cccccc" };
+            case "Steel":
+                return { color: " #999999" };
+            default:
+                return {};
+        }
+    }
+
+    /* Capitalizes the first letter of the type, might remove later */
+    const capitalize = (str) => {
+        if(str === ""){return "";}
+        return str.charAt(0).toUpperCase() + str.slice(1);
+    }
+
```

Comment:
can remove later, the test backend has lower right now but once I update it to the actual backend it'll work fine